### ISSUE TITLE
Add "double tap on map" to activate "follow my location"

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#16045]
Request on support, ticket 843982:

"Double tap map to center on location.
I know there is the circle icon for this on the top of the map. I use cgeo often on my bike, mounted on the handle bar.

While cycling it is almost impossible to hit the center icon, but very important, to see where you are and where you need to go.

A quick way to center on position (eg by double tapping the map) would be very helpful and improve safety."

**Comment by SammysHP** [comment]
Double tap is used to zoom in already. I guess many users would be unhappy about changing the behavior.

Let's ignore the proposed solution for now and focus on the problem: The follow me button is hard to hit in some situations. Likely this is true also for other buttons.

A common approach is to offer adaptive views depending on the situations. The multimedia system of modern cars will often switch the view depending on speed, with all features available only for low speeds, while high speeds offer only some large, important buttons to the user. I don't think we can directly apply this to c:geo, but having a simplified off-road layout that can be activated manually might work.

**Comment by eddiemuc** [comment]
I agree with @SammysHP in that we shouldn't replace the well-known meaning of double-tap as "zoom" in c:geo.

I suggest to look how other map apps are solving this. I looked into Locus and Google Maps who both simply offer a more prominent (means: not so far at the screen edge) place for the "center" button.

**Comment by moving-bits** [comment]
> I agree with @SammysHP in that we shouldn't replace the well-known meaning of double-tap as "zoom" in c:geo.

Yes, we should not change the "double tap=zoom" meaning, I'll rename the issue.

> I suggest to look how other map apps are solving this. I looked into Locus and Google Maps who both simply offer a more prominent (means: not so far at the screen edge) place for the "center" button.

We have discussed some ideas for different positions in #15774.

I'm not sure, though, if this will help the TO. To my understanding the problem for the TO is more about how to hit such a small button while cycling.

**Comment by moving-bits** [comment]
from today's dev meeting: 
move "follow me" icon to map first (see separate issue), then let's see if TO's issue still persists

**Comment by moving-bits** [comment]
nothing has come up on support about this request anymore, so let's close here
